### PR TITLE
add read_policies to share reader and writer workspace roles

### PR DESCRIPTION
**Pull request by dvoet**
Ticket: <Link to Jira ticket>
<Put notes here to help reviewer understand this PR>

---

**PR checklist**

- [x] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

**Review by MatthewBemis — APPROVED** [review]

File: src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SamRoutes.scala [hunk 1]
```diff
@@ -48,6 +48,7 @@ abstract class SamRoutes(val resourceService: ResourceService, val userService:
         case Complete(resp) =>
           val logLevel: LogLevel = resp.status.intValue / 100 match {
             case 5 => Logging.ErrorLevel
+            case 4 => Logging.InfoLevel
```
Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SamRoutes.scala:51]: unrelated change that I made to rawls to allow us to select to log 4xx level errors without swamping the logs with successful responses too

**Review by gpolumbo-broad — APPROVED** [review]

**Comment by dvoet** [comment]
jenkins retest because the first test results are gone already
